hpy/tools/autogen/conf.py

```diff
@@ -139,4 +139,7 @@
     'HPyBool_FromBool': 'PyBool_FromLong',
     'HPy_Compile_s': None,
     'HPy_EvalCode': 'PyEval_EvalCode',
+    'HPyContextVar_New': 'PyContextVar_New',
```

Comment:
this one and the one for `HPyContextVar_Set` is not necessary; autogen will do that automatically

Reply:
OK